sherpa/utils/random.py

```diff
@@ -120,7 +147,11 @@ def random(rng):
     return rng.random()
 
 
-def uniform(rng, low, high, size=None):
+def uniform(rng: Optional[RandomType],
+            low: float,
+            high: float,
+            size=None
```

Comment:
I perhaps should have added a note about why I have not added a type for size. This is because it would have to be something like

```
    size: Optional[int | Sequence[int] | ndarray]
```

and I didn't think it helped to say something like this.

Reply:
Comment has now been added.